src/app/shared/components/cluster-summary/template.html

```diff
@@ -116,6 +116,9 @@
       <km-property-boolean label="OPA Integration"
                            [value]="cluster.spec.opaIntegration?.enabled">
       </km-property-boolean>
+      <km-property-boolean label="Konnectivity"
+                           [value]="cluster.spec.clusterNetwork?.konnectivityEnabled">
+      </km-property-boolean>
```

Comment:
Small thing, but since we place the checkbox in the wizard in the `Cluster Network` section, I think it would make sense to also put it close to it in the summary view. (Or other way round: put konnectivity to the checkboxes on the right. Whatever makes more sense for the user.)

Step:
![wizard1](https://user-images.githubusercontent.com/19547196/132359460-3fc96336-1431-4682-bf51-876df00b205c.PNG)

Summary:
![wizard2](https://user-images.githubusercontent.com/19547196/132359466-ed2ef54c-8f24-4285-9b04-cf11d2c3d002.PNG)

Reply:
100%. moving...